index.js

```diff
@@ -5,8 +5,10 @@ async function createGitHubDeployment(context) {
   const pr_branch = pull_request.head.ref;
 
   context.log(`A PR was ${payload.action}. Branch: `, pr_branch);
+  const task = payload.action === 'reopened' ? 'redeploy' : 'deploy';
```

Comment:
I think I might actually want to change the convention here: Right now there are two tasks. Roughly,

* `deploy` happens on the `open` and `synchronize` events. It will create a new database if one does not exist. If one does exist, the new shard will use it
* `redeploy` happens on the `reopen` event. It will always destroy the database if one exists before launching the new shard.

But really, `open` and `reopen` seem more similar to me and have the semantics of "I expect a brand new database. So maybe putting those events together makes more sense. We could have 

* `deploy` happens on `open` and `reopen` events. It also destroys the existing db if there is one.
* `update` happens only on `synchronize` events. It never tries to destroy an existing db, but will create a new db if none exists.

What do you think?

Reply:
I agree - `open` and `reopen` would do the `create` (or `create_deployment`) task, and `synchronize` would do the `update` (or `update_deployment`) task.

Reply:
Or maybe `create_review_app` and `update_review_app`

Reply:
I like `create_review_app` and `update_review_app` 👍